Formula/convco-cli.rb

```diff
@@ -0,0 +1,32 @@
+class ConvcoCli < Formula
+  desc "Conventional commits, changelog, versioning, validation"
+  homepage "https://convco.github.io"
+  url "https://github.com/convco/convco/archive/refs/tags/v0.4.0.tar.gz"
+  sha256 "88c92b175163c8847da7dd201d32106c51ff85e2c992c2a3ff67e29ecbe57abb"
+  license "MIT"
+  head "https://github.com/convco/convco.git", branch: "master"
+
+  livecheck do
+    url "https://github.com/convco/convco/releases/latest"
+    regex(%r{href=.*?/tag/v?(\d+(?:\.\d+)+)["' >]}i)
+  end
+
+  # used by git2 crate
+  depends_on "cmake" => :build
```

Comment:
Let's do
```ruby
depends_on "libgit2"
```
instead of building another one here.

Reply:
Revisiting this (see build issues in another comment). After reading [this information from libgit2](https://github.com/rust-lang/git2-rs#version-of-libgit2) I'm not so sure how useful it is to have the dynamically linked library installed since the rust crate would build its own version either way (and in this case, with the zlib-ng feature).

Reply:
Edit: I am now patching the build process to avoid using the embedded `libgit2` library.